akka-remote/src/main/scala/akka/remote/artery/compress/InboundCompressions.scala

```diff
@@ -77,7 +79,7 @@ private[remote] abstract class InboundCompression[T >: Null](
   val heavyHitters:   TopHeavyHitters[T],
   convertKeyToString: T ⇒ String) { // TODO avoid converting to string, in order to use the ActorRef.hashCode!
 
-  val log = Logging(system, "InboundCompressionTable")
+  lazy val log = Logging(system, getClass.getSimpleName)
```

Comment:
I don't we should use short names for the loggers. We should stick normal (slf4j) conventions, i.e. use full class name.